### PR TITLE
Fix #12466 adding viewport must solve mobile usability problem   

### LINKED ISSUE #12466: Add Viewport to supabase.com
image

### PR DESCRIPTION
**Pull request by dsukruth**
## What kind of change does this PR introduce?
 

Bug fix,

## What is the current behavior?

Please link any relevant issues here.
#12466
## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context
Must fix the mobile usability problem
Add any other context or screenshots.

### CODE REVIEW

**Review by kiwicopple — APPROVED** [review]
that was fast, thanks @dsukruth !

**Review by kiwicopple — CHANGES_REQUESTED** [review]
Actually - i realised you've fixed one of the "examples", rather than the `supabase.com` repo (which is under `/apps/..`)

**Comment by dsukruth** [comment]
Haha! my bad, working on it

File: apps/docs/components/DocSearch.tsx [hunk 1]
```diff
@@ -48,6 +48,7 @@ export function SearchProvider({ children }: any) {
     <>
       <Head>
         <link rel="preconnect" href={`https://${APP_ID}-dsn.algolia.net`} crossOrigin="true" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"></meta>
```
Comment (kiwicopple) [inline apps/docs/components/DocSearch.tsx:51]: I don't know if this one is required - looks like it's just a search component

File: apps/docs/components/Favicons.tsx [hunk 2]
```diff
@@ -25,6 +25,7 @@ const Favicons = () => {
       <link rel="manifest" href={`${basePath}/favicon/site.webmanifest`} />
       {/* <link rel="mask-icon" href={`${basePath}/favicon/safari-pinned-tab.svg`} color="#333333" /> */}
       <link rel="shortcut icon" href={`${basePath}/favicon/favicon.ico`} />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0"></meta>
```
Comment (kiwicopple) [inline apps/docs/components/Favicons.tsx:28]: I think we can remove from here too

File: apps/docs/layouts/DefaultLayout.tsx [hunk 3]
```diff
@@ -21,7 +21,7 @@ const Layout: FC<Props> = (props: Props) => {
       <Head>
         <title>{props.meta?.title} | Supabase Docs</title>
         <meta name="description" content={props.meta?.description} />
-        <meta content="width=device-width, initial-scale=1" name="viewport" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
```
Comment (kiwicopple) [inline apps/docs/layouts/DefaultLayout.tsx:24]: nice - this one is good 👍

File: apps/docs/components/DocSearch.tsx (hunk 1 above)
Comment (kiwicopple) [inline apps/docs/components/DocSearch.tsx:51]:
```suggestion
```

File: apps/docs/components/Favicons.tsx (hunk 2 above)
Comment (kiwicopple) [inline apps/docs/components/Favicons.tsx:28]:
```suggestion
```

**Review by kiwicopple — APPROVED** [review]

File: examples/user-management/sveltekit-user-management/src/app.html [hunk 4]
```diff
@@ -3,7 +3,7 @@
 	<head>
 		<meta charset="utf-8" />
 		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
-		<meta name="viewport" content="width=device-width" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (MildTomato) [inline examples/user-management/sveltekit-user-management/src/app.html:6]: this one added by mistake ?

Comment (dsukruth) [reply examples/user-management/sveltekit-user-management/src/app.html:6]: initial-scale=1.0 part sets the initial zoom level when the page is first loaded by the browser.

Comment (MildTomato) [reply examples/user-management/sveltekit-user-management/src/app.html:6]: I meant is it a mistake because it's in our examples directory 😅

Comment (dsukruth) [reply examples/user-management/sveltekit-user-management/src/app.html:6]: haha! yes, it is a mistake,wrong directory 😅

**Pull request merged**